test/testMethods.ts

```diff
@@ -91,18 +91,27 @@ module TestMethods {
     assert.strictEqual(height, String(heightExpected), "height: " + message);
   }
 
-  export function assertEntitiesEqual(actual: Plottable.Plots.Entity, expected: Plottable.Plots.Entity,
-      msg: string) {
+  export function assertEntitiesEqual(
+    actual: Plottable.Entity<Plottable.Component>,
+    expected: Plottable.Entity<Plottable.Component>,
+    msg: string) {
```

Comment:
I prefer double indentation here (add 2 more spaces) to distinguish from the body of the method in some way)